### DISCUSSION TITLE
Is there any ways to bind configs to pojo like spring ConfigurationProperties ??

### DISCUSSION THREAD

**Discussion by iammohan01** [playframework/playframework#12633] [Q&A]
In spring we can simply annotate a config class with @ConfigurationProperties ?? is there any ways to bind and inject configs props into pojo in playframwork ??

**Comment by will-sargent-eero** [comment]
You can use Dagger 2 or Guide to inject configuration.  https://www.playframework.com/documentation/2.8.x/JavaDependencyInjection
